### DISCUSSION TITLE
Issues with OSPSuite.FuncParserNative.dll when downloading OSP

### DISCUSSION THREAD

**Discussion by KaiWiater** [Open-Systems-Pharmacology/Forum#1867] [Questions & Problems] [answered]
Hi!
I recently donwloaded the program and when I try to open any file, I get an error message with problem with OSPSuite.FuncParserNative.dll.
I attach a screenshot of the message. Does anyone have a solution? Thanks!
<img width="1440" alt="Screenshot 2024-11-08 at 08 53 19" src="https://github.com/user-attachments/assets/fb803575-3f0f-4319-a8b6-4ea53685b838">

**Comment by Yuri05** [comment]
What is your operating system? Windows 11?
Did you install the full Suite Setup or are you using a portable version?

**Comment by rwmcintosh** [comment, marked as answer]
Install the latest [VC Redist](https://aka.ms/vs/17/release/vc_redist.x64.exe)
